internal/runners/export/config/config.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+)
+
+type Config struct {
+	out output.Outputer
+}
+
+type ConfigParams struct {
+	Filter Filter
+}
+
+type primeable interface {
+	primer.Outputer
+}
+
+func New(prime primeable) *Config {
+	return &Config{prime.Output()}
+}
+
+func (c *Config) Run(cmd *captain.Command, params *ConfigParams) error {
+	output := map[string]string{
+		Dir.String(): config.ConfigPath(),
+	}
+
+	filteredOutput := map[string]string{}
+	if params.Filter.terms == nil {
+		filteredOutput = output
+	}
```

Comment:
nit: I would invert this so it flows a little better, eg. `filteredOutput := output` is the initial definition, then the empty map and for loop happen inside an `!= nil` conditional.